corehq/apps/reports/standard/tableau.py

```diff
@@ -76,12 +77,12 @@ def tableau_server_response(self):
                 self.context.update({'ticket': tabserver_response.content.decode('utf-8')})
                 return super().view_response
             else:
-                self.context.update({"failure_message": "Tableau trusted authentication failed"})
+                self.context.update({"failure_message": _("Tableau trusted authentication failed")})
```

Comment:
So these ones ought to be `ugettext`, because this code is called when generating a response, so there **is** a request and a user available.